build/yamls/antrea-eks.yml

```diff
@@ -419,6 +432,8 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: spec.nodeName
+        - name: API_PORT
+          value: "10349"
```

Comment:
why we need this?

Reply:
Replied in `pkg/antctl/raw/bundle/command.go` thread.